ui-speedspacechart/src/components/const.ts

```diff
@@ -63,6 +65,13 @@ export const LAYERS_SELECTION: Array<keyof Store['layersDisplay']> = [
   'speedLimitTags',
 ];
 
+// Colors
+
+export const BLACK = '#000000';
+export const GREY_50 = '#797671';
+export const GREY_80 = '#312e2b';
+export const LIGHT_BLUE = '#2170b9';
```

Comment:
we usually use rgba() instead of hexa in OSRD-UI, is there a reason to use hexa in this pr?

Reply:
I took example from the space time chart. 

Kind of related. Do you have a suggestion for this issue: https://github.com/OpenRailAssociation/osrd-ui/pull/455/files/9fab2503afcb3d252edc2304a4f9a2754cfaf383#r1761472603 ?